packages/swapper/src/swappers/ChainflipSwapper/swapperApi/getTradeQuote.ts

```diff
@@ -164,20 +165,22 @@ export const getTradeQuote = async (
       case CHAIN_NAMESPACE.Utxo: {
         const sellAdapter = deps.assertGetUtxoChainAdapter(sellAsset.chainId)
         const publicKey = (input as GetUtxoTradeQuoteInput).xpub!
+        // TODO(gomes): double check dis correct
         return await getUtxoTxFees({
           sellAmountCryptoBaseUnit: sellAmount,
           sellAdapter,
           publicKey,
         })
       }
 
-      case CHAIN_NAMESPACE.Solana: {
+      // TODO(gomes): Cosmos SDK too
+      case CHAIN_NAMESPACE.CosmosSdk:
```

Comment:
Will this ever be the case? No idea what `CosmosSdk` is, but chainflip doesnt support any cosmos chains (right now)